lib/types.ts

```diff
@@ -270,3 +270,21 @@ export type TemplateDatetimePickerAction = {
   max?: string;
   min?: string;
 };
+
+export type Size = {
+  width: number;
+  height: number;
+};
+
+export type RichMenuArea = {
+  bounds: ImageMapArea;
```

Comment:
I think it's better renaming `ImageMapArea` into `Area`, as it's now used not only in `ImageMap`.